bait-files/src/main/webapp/WEB-INF/Error404.jsp

```diff
@@ -0,0 +1,15 @@
+<%@ page language="java" contentType="text/html; charset=ISO-8859-1"
+    pageEncoding="ISO-8859-1"%>
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN"
+    "http://www.w3.org/TR/html4/loose.dtd">
+<html>
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=ISO-8859-1">
+<title>404 Error - Page Not Found</title>
+</head>
+<body>
+    <center>
+        <h1>Sorry, the page you requested were not found.</h1>
```

Comment:
very small change but should be "was not found"

Reply:
Removed